inscriptions/ISic001162.xml

```diff
@@ -192,22 +197,17 @@
         <body>
             <div type="edition" xml:space="preserve" subtype="PHI" n="PHI140647">
                 <ab>
-                    <lb n="1"/>
-                    <supplied reason="lost">․․</supplied>όδωρε
-                    <lb n="2"/>
-                    <supplied reason="lost">․․</supplied>νίου
-                    <lb n="3"/>
-                    <supplied reason="lost">Βρ</supplied>ίττιε
-                    <lb n="4"/>
-                    <supplied reason="lost">χα</supplied>ῖρε.
-                    <lb n="5"/>
-                </ab>
+                    <lb n="1"/><gap reason="lost" extent="unknown" unit="character"/><name seg="F">όδωρε</name>
```

Comment:
right idea, but the seg is a separate element, child of name:
`<name><seg part="F">όδωρε</seg></name>`